modules/go-compatibility-and-support-matrix.adoc

```diff
@@ -15,6 +15,7 @@ In the table, features are marked with the following statuses:
 |*OpenShift GitOps* 7+|*Component Versions*|*OpenShift Versions*
 
 |*Version*|*kam*    |*Helm*  |*Kustomize*|*Argo CD*|*ApplicationSet*|*Dex*     |*RH SSO* |
+|1.6.0    |0.0.46 TP|3.8.1 GA|4.4.1 GA   |2.4.5 GA |0.4.1 GA        |2.30.3 GA |7.5.1 GA |4.8-4.10
```

Comment:
ApplicationSets has been merged to Argo CD core. There will not be a separate version for ApplicationSet. Also, we need to mention ApplicationSet has gone GA.